inc/namespace.php

```diff
@@ -477,7 +477,13 @@ function load_db() {
  * @return string
  */
 function get_main_site_url( string $path = '' ) : string {
-	return get_site_url( get_main_site_id( get_main_network_id() ), $path );
+	static $main_site_url = '';
```

Comment:
I think you can do this without assigning an initial value too, but the end result is the same. Just feels a little better to do a boolean check against `null` or something that returns a boolean.
```suggestion
	static $main_site_url;
```